src/lib/create-utils.js

```diff
@@ -85,6 +86,33 @@ exports.enhanceReadmeWithLocalSuffix = async (readmePath, manifestPromise) => {
     }
 };
 
+/**
+ * Goes to the Actor directory and creates INPUT.json file from the input schema prefills.
+ * @param {string} actFolderDir
+ */
+exports.createInputFromSchema = async (actFolderDir) => {
+    try {
+        const currentDir = process.cwd();
+        process.chdir(actFolderDir);
+        const { inputSchema } = await readInputSchema();
+
+        if (inputSchema) {
+            const input = Object.keys(inputSchema.properties).reduce((acc, key) => {
+                const { prefill } = inputSchema.properties[key];
+                if (prefill) acc[key] = prefill;
```

Comment:
I'm not sure if it's sufficient exception. There still can be required properties without prefill, right? Then the run would fail. I'd add some default values for that (empty string, 0, first item in enum etc.)